test/runtests.jl

```diff
@@ -9,7 +9,13 @@ using Test
             A = rand(T,sizes...)
             Ac = zfp_compress(A)
             Ad = zfp_decompress(Ac)
-            @test Ad == A
+
+            A_view = reshape(view(A, :), size(A))
+            Ac_view = zfp_compress(A_view)
+            Ad_view = zfp_decompress(view(Ac,:))
```

Comment:
This test is good because `view` changes the type of the input array, but right now it would still have unit stride (contiguous in memory), what we should test here is a non-unit stride, for example created by a view onto every second element of the array. 
```julia
julia> A = rand(10)
julia> strides(A)
(1,)

julia> strides(view(A,:))
(1,)

julia> strides(view(A,1:2:10))
(2,)
```
Theoretically this information should be passed on with the ccall, but we'd need to test that!